XPlanning/data/mobilerobot/maps/GHC7-map0.json

```diff
@@ -198,7 +198,6 @@
         "y": 5.33
       },
       "connected-to": [
-        "L26",
```

Comment:
Removing a connection in the map. Need to remove from 'connected-to' at both ends.